res/manual/src/help.md

```diff
@@ -0,0 +1,6 @@
+# Additional Help
+
+Using Anthem effectively can sometimes require sophisticated proof outlines.
+This section has a couple of example lemmas that may help.
+
+To report bugs or unexpected behavior in Anthem, please email zachhansen@unomaha.edu or tobias.stolzmann@gmail.com.
```

Comment:
Three things:
1. I'd put our contact information into a dedicated section. (How to name it?)
2. I'd use links [like this](https://github.com/mattcone/markdown-guide/blob/master/_basic-syntax/links.md#urls-and-email-addresses) here.
3. We should think about a Thanks section where we list all current and past contributers like Vladimir, Patrick, Jan, Torsten, Yuliya, Jorge, ...